app/src/main/java/com/hadiyarajesh/notex/reminder/notification/NotificationHelper.kt

```diff
@@ -0,0 +1,72 @@
+package com.hadiyarajesh.notex.reminder.notification
+
+import android.app.PendingIntent
+import android.app.PendingIntent.FLAG_IMMUTABLE
+import android.content.Context
+import android.content.Intent
+import androidx.core.app.NotificationCompat
+import androidx.core.app.NotificationManagerCompat
+import com.hadiyarajesh.notex.MainActivity
+import com.hadiyarajesh.notex.R
+import javax.inject.Inject
+import kotlin.random.Random
+
+class NotificationHelper @Inject constructor() {
```

Comment:
what are we injecting here?

Constructor() is fully empty

Reply:
because it is injected in worker